### DISCUSSION TITLE
How to disable file icons?

### DISCUSSION THREAD

**Discussion by morzel85** [ibhagwan/fzf-lua#1631] [Q&A] [answered]
Is there some option to get rid of file icons? I'm doing just fine without them in terminal fzf and would like to have my fzf-lua output a bit more plaintext too...

**Comment by ibhagwan** [comment, marked as answer]
Use max-perf profile or set defaults.file_icons=false.

Btw, this is rather common see OPTIONS.md, issues, etc.

**Comment by morzel85** [comment]
Thanks for super quick answer! I've switched to this lazy config and got what I wanted :)

```
  {
    'ibhagwan/fzf-lua',
    config = function()
      require('fzf-lua').setup { 'max-perf' }
    end,
  },
```
